docs/pages/installation/amazon-ecs.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Installing Teleport on Amazon ECS using Terraform
+description: How to install and run Teleport agents on Amazon ECS
+labels:
+ - platform-wide
+ - reference
+---
+
+Using Amazon ECS and Terraform allows you to easily deploy and manage Teleport Agents that are closer to your AWS resources.
+
+This flow is ideal when you want to auto discover and secure access to your EKS clusters, RDS databases or other supported AWS resources.
+
+## How it works
+Amazon ECS provides a convenient way to deploy Teleport agents closely located to your AWS resources, allowing access even if they are not publicly accessible.
+With Terraform, you can easily deploy and keep the Teleport agent up to date.
+
+## Prerequisites
+The following set up is required:
+- install [`terraform`](https://developer.hashicorp.com/terraform/install)
+- [configure AWS credentials](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication-and-configuration)
+- a Teleport [IAM Join Token](../reference/join-methods.mdx#aws-iam-role-iam) so that your agent can securely join the Teleport cluster
+
+## Step 1/3. Download the terraform module
+
+Download the source code for our example:
+```code
+$ git clone https://github.com/gravitational/teleport -b branch/v(=teleport.major_version=)
+$ cd teleport/examples/aws/terraform/ecs-agent
```

Comment:
Which PR introduces this module?

Reply:
Note to self: remember to `git add` the most important bits 😅 

I'll add it in the next commit.